apps/analysis/views.py

```diff
@@ -165,7 +165,7 @@ def get(self, request, version=None):
         options = [
             {
                 'key': tag.value,
-                'value': tag.name.title()
+                'value': tag.label,
```

Comment:
Please fix the test case as well.


![2021-07-22-091211](https://user-images.githubusercontent.com/7059255/126586246-f691ca8b-2d25-49c6-aefd-bd3e257917d0.png)

Change `name.title()` to `label` here
https://github.com/the-deep/server/blob/10ba1dca74821210d5532ce4a6f7ecb4d5450881/apps/analysis/tests/test_apis.py#L1130-L1143

Reply:
Okay sir!